### PR TITLE
[MRG] Added RTD link in README

### PR DESCRIPTION
**Pull request by NicolasHug**
I've also added 
@rth @jnothman @ogrisel @GaelVaroquaux as admins of the RTD repo (couldn't find you @amueller?).

This should be enough for RTD to be triggered everytime a new commit is pushed to master, else one of you guys will have to setup the hook manually.

### CODE REVIEW

**Comment by amueller** [comment]
I'm @t3kcit I think.

**Comment by amueller** [comment]
Can you make the empty sections show up? They don't seem to show on rtd :-/

**Comment by rth** [comment]
I guess it would have been nice to have `enhancement-proposals.scikit-learn.org` instead of the current `scikit-learn-enhancement-proposals.readthedocs.io` with [the custom domain support on RTD](https://docs.readthedocs.io/en/latest/custom_domains.html#custom-domain-support), but it's not too critical.

**Comment by amueller** [comment]
@ogrisel can probably do that for us? I also think that it would be nicer. I don't think there is many links to that site yet, so I wouldn't worry about changing it.

**Comment by amueller** [comment]
And if you make it headers instead of toctree titles?

**Comment by NicolasHug** [comment]
I'm not sure what you mean?

I've tried different things and I think the easiest way is just to create empty files for the titles to show up. We can remove them as soon as we add sleps under those sections.

File: accepted.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Accpeted SLEPs
```
Comment (amueller) [inline accepted.rst]: typo

**Comment by amueller** [comment]
I think it's fine to merge as-is, we can change the url once @ogrisel can change the DNS ;)

**Comment by amueller** [comment]
that wasn't rendered on rtd for some reason, right?

**Comment by NicolasHug** [comment]
Yeah I just opened  #11 which should solve this

**Comment by amueller** [comment]
Sorry, I mean the change in the TOC, the empty sections don't show up on rtd

**Comment by NicolasHug** [comment]
Oh right! Should be fine now, I manually ran the RTD build.
On of you guys admins will have to setup the hook if you want this to be automatic.

(#11 is still worth merging maybe)

**Comment by amueller** [comment]
link for setting up the hook?

**Comment by NicolasHug** [comment]
lol no worries [that](https://docs.readthedocs.io/en/latest/webhooks.html#webhook-creation) should be it

**Pull request merged**